### DISCUSSION TITLE
Error code 9009

### DISCUSSION THREAD

**Discussion by TheMechatronicsEngineer** [atom/atom#23547] [Q&A]
Hi all
I'm new to Atom.
Do you know what thjis error mean and what can I do to resolve the issue?
Thanks 
![immagine](https://user-images.githubusercontent.com/98771009/151942982-3330c861-4614-43e0-a1d2-23514c69ac4d.png)

**Comment by icecream17** [comment]
I think you can try looking at https://github.com/michaelaquilina/python-tools/issues/new
![image](https://user-images.githubusercontent.com/58114641/151992903-b3515056-1d1b-46a2-bd1a-6d56169fe78a.png)

<details>

<summary>Although this has actually happened a bunch of times: here are all the duplicates</summary>

https://github.com/MichaelAquilina/python-tools/issues/160
https://github.com/MichaelAquilina/python-tools/issues/167
https://github.com/MichaelAquilina/python-tools/issues/170
https://github.com/MichaelAquilina/python-tools/issues/172
https://github.com/MichaelAquilina/python-tools/issues/175
https://github.com/MichaelAquilina/python-tools/issues/177
https://github.com/MichaelAquilina/python-tools/issues/184
https://github.com/MichaelAquilina/python-tools/issues/189
https://github.com/MichaelAquilina/python-tools/issues/192
https://github.com/MichaelAquilina/python-tools/issues/198
https://github.com/MichaelAquilina/python-tools/issues/199
https://github.com/MichaelAquilina/python-tools/issues/200
https://github.com/MichaelAquilina/python-tools/issues/206
https://github.com/MichaelAquilina/python-tools/issues/221
https://github.com/MichaelAquilina/python-tools/issues/224
https://github.com/MichaelAquilina/python-tools/issues/229
https://github.com/MichaelAquilina/python-tools/issues/230
https://github.com/MichaelAquilina/python-tools/issues/231
https://github.com/MichaelAquilina/python-tools/issues/233
https://github.com/MichaelAquilina/python-tools/issues/234
https://github.com/MichaelAquilina/python-tools/issues/241
https://github.com/MichaelAquilina/python-tools/issues/244
https://github.com/MichaelAquilina/python-tools/issues/245
https://github.com/MichaelAquilina/python-tools/issues/247
https://github.com/MichaelAquilina/python-tools/issues/248
https://github.com/MichaelAquilina/python-tools/issues/249
https://github.com/MichaelAquilina/python-tools/issues/250
https://github.com/MichaelAquilina/python-tools/issues/251
https://github.com/MichaelAquilina/python-tools/issues/252
https://github.com/MichaelAquilina/python-tools/issues/262
https://github.com/MichaelAquilina/python-tools/issues/263
https://github.com/MichaelAquilina/python-tools/issues/272
https://github.com/MichaelAquilina/python-tools/issues/274
https://github.com/MichaelAquilina/python-tools/issues/281
https://github.com/MichaelAquilina/python-tools/issues/282

</details>

For now the only solution is to:

{Wait for the bug to be fixed | Fix the bug} and {wait for the fix to be published | publish a new package with the fix} and finally install the updated/new package.
